packages/contracts-bedrock/package.json

```diff
@@ -18,8 +18,8 @@
     "build:forge": "forge build",
     "build:differential": "go build -o ./scripts/differential-testing/differential-testing ./scripts/differential-testing",
     "build:fuzz": "(cd test-case-generator && go build ./cmd/fuzz.go)",
-    "generate:addresses": "tsx scripts/generate-addresses.ts",
-    "generate:addresses:check": "pnpm generate:addresses && git diff --exit-code ./addresses.json && git diff --exit-code ./addresses.json",
+    "generate:addresses": "pnpm --filter=@eth-optimism/contracts-ts generate:addresses $PWD/addresses.json",
+    "generate:addresses:check": "pnpm generate:addresses && git diff --exit-code ./addresses.json",
```

Comment:
Since this script doesn't have a build step it can run without adding @eth-optimism/contracts-ts as a dep
If I added the script to .bin it would have even been able to reference it directly in path without needing to call with pnpm. I chose pnpm because it's explicit though